lib/prmd/commands/init.rb

```diff
@@ -9,9 +9,21 @@ module Generate
     # Creates a default Prmd::Generator using default templates
     #
     # @return [Prmd::Generator]
-    def self.make_generator
+    def self.make_generator(options)
       base = Prmd::Template.load_json('init_default.json')
-      template = Prmd::Template.load_template('init_resource.json.erb', '')
+      template_name = options.fetch(:template) do
+        abort 'render: Template was not provided'
```

Comment:
I think that if you supplied nothing, this abort would be hit if I'm not mistaken? I think that would prevent the default below from ever being reached, so wanted to double check.